examples/Bunch/include/simulation_defines/param/fieldBackground.param

```diff
@@ -0,0 +1,222 @@
+        HINLINE FieldBackgroundE( const float3_64 unitField ) : unitField(unitField),
+        
+            twtsFieldE( 30.0e-6,  /* focus_y [m], the distance to the laser focus in y-direction */
+                /* wavelength [m] */
+                0.8e-6,
+                /* pulselength [s], sigma of std. gauss for intensity (E^2) */
+                10.0e-15 / 2.3548200450309493820231386529194,
+                /* w_x [m], cylindrically focused spot size */
+                5.0e-6,
+                /* w_y [m] */
+                0.01,
+                /* interaction angle between TWTS laser propagation vector and the y-axis [rad] */
+                60. *(PI/180.),
+                /* propagation speed of overlap [speed of light]. */
+                1.0,
+                /* manual time delay [s] if auto_tdelay is false */
+                ( 39.3e-6 )/(::picongpu::SI::SPEED_OF_LIGHT_SI),
+                /* Should PIConGPU automatically choose a suitable time delay? [true/false] */
+                false)
+        {}
+
+        /** Specify your background field E(r,t) here
+         *
+         * \param cellIdx The total cell id counted from the start at t=0
+         * \param currentStep The current time step */
+        HDINLINE float3_X
+        operator()( const DataSpace<simDim>& cellIdx,
+                    const uint32_t currentStep ) const
+        {
+            /* example 1: TWTS background pulse */
+            /** unit: meter */
+            const double WAVE_LENGTH_SI = 0.8e-6;
+            
+            /** UNITCONV */
+            /* const double UNITCONV_Intens_to_A0 = ::picongpu::SI::ELECTRON_CHARGE_SI
+                * ::picongpu::SI::ELECTRON_CHARGE_SI * 2.0 * WAVE_LENGTH_SI * WAVE_LENGTH_SI
+                / (4.0 * PI * PI * ::picongpu::SI::ELECTRON_MASS_SI
+                * ::picongpu::SI::ELECTRON_MASS_SI * ::picongpu::SI::SPEED_OF_LIGHT_SI
+                * ::picongpu::SI::SPEED_OF_LIGHT_SI * ::picongpu::SI::SPEED_OF_LIGHT_SI
+                * ::picongpu::SI::SPEED_OF_LIGHT_SI * ::picongpu::SI::SPEED_OF_LIGHT_SI
+                * ::picongpu::SI::EPS0_SI); */
+            const double UNITCONV_A0_to_Amplitude_SI = -2.0 * PI / WAVE_LENGTH_SI
+                * ::picongpu::SI::ELECTRON_MASS_SI * ::picongpu::SI::SPEED_OF_LIGHT_SI
+                * ::picongpu::SI::SPEED_OF_LIGHT_SI / ::picongpu::SI::ELECTRON_CHARGE_SI;
+
+            /** unit: W / m^2 */
+            /* const double _PEAK_INTENSITY_SI = 3.4e19 * 1.0e4; */
+            /** unit: none */
+            /* const double _A0  = _PEAK_INTENSITY_SI * UNITCONV_Intens_to_A0; */
+
+            /** unit: none */
+            const double _A0  = 1.0;
+
+            /** unit: Volt /meter */
+            const float3_64 invUnitField = float3_64(1.0/unitField[0],
+                             1.0/unitField[1],
+                             1.0/unitField[2]);
+            
+            const float3_X AMPLITUDE_SI = precisionCast<float_X>(
```

Comment:
please use the name `amplitude_SI` or `amplitude_si` instead of `AMPLITUDE_SI`
A typical C/C++ code convention is to write only defines with upper case letters.

In PIConGPU we sometimes use *upper case letters* for global param/unitless variables inside namespace.